src/d_iwad.c

```diff
@@ -28,22 +28,22 @@
 #include "m_misc.h"
 
 static const iwad_t iwads[] = {
-    {"doom2.wad",     doom2,      commercial,   vanilla,  "Doom II"                   },
-    {"plutonia.wad",  pack_plut,  commercial,   vanilla,
-     "Final Doom: Plutonia Experiment"                                                },
-    {"tnt.wad",       pack_tnt,   commercial,   vanilla,  "Final Doom: TNT: Evilution"},
+    {"doom2.wad",     doom2,      commercial,   "doom",     "Doom II"                   },
```

Comment:
How about "doom2-all" for doom2.wad, tnt.wad and plutonia.wad?

Reply:
Hm, and "doom-all" then remains for Doom 1 only?

Reply:
And how about freedoom2.wad?

Reply:
> Hm, and "doom-all" then remains for Doom 1 only?

Yes. It makes sense because there are many variants of Doom 1: shareware, registered, ultimate.

Reply:
> And how about freedoom2.wad?

I don't know about specific freedoom2 content. But maybe we should do it for consistency.

Reply:
The whole point of this request was that "doom-all" was not autoloaded for freedoom IWADs. It's tricky.

Reply:
And it makes sense. Sprite or sound replacements won't work for both doom2.wad and freedoom2.wad.

Reply:
What are the use cases for autoloaded wad files for Doom 1 but not Doom 2 and vice versa. I can imagine people load music packs, sprite replacements or custom palettes, but these work equally well for both Doom versions, or not?

Reply:
example:
"all" for huds and other cross-iwad mods
"doom-all" for perkristian sfx or the soundbulb mod
"doom1-all" for D1SPFX20 and its dehacked
"doom2-all" for D2SPFX20 and its dehacked